src/site/stages/build/plugins/check-broken-links/helpers/getErrorOutput.js

```diff
@@ -0,0 +1,22 @@
+function getErrorOutput(brokenPages) {
+  const brokenLinkCount = brokenPages.reduce(
+    (sum, page) => sum + page.linkErrors.length,
+    0,
+  );
+  const separator = '\n\n---\n\n';
+  const header = `${separator}There are ${brokenLinkCount} broken links!${separator}`;
+
+  const body = brokenPages
+    .map(brokenPage => {
+      let pageChunk = `There are ${
+        brokenPage.linkErrors.length
+      } broken links on ${brokenPage.path}:\n\n`;
+      pageChunk += brokenPage.linkErrors.map(link => link.html).join('\n');
+      return pageChunk;
+    })
+    .join(separator);
+
+  return header + body + separator;
+}
+
+module.exports = getErrorOutput;
```

Comment:
This takes an array of pages containing broken links and formats it into helpful console output. The output could still be improved to be "smart" and say things like "There are X broken links to this one page - could that page have been unpublished by mistake?" but it's still a big improvement for now, by at least printing the HTML containing the bad link -

```

---

There are 79 broken links!

---

There are 1 broken links on 404.md:

<a href="/disability/how-to-file-claim/">How do I apply for disability benefits?</a>

---

There are 1 broken links on maintenance.md:

<a href="/disability/how-to-file-claim/">How do I apply for disability benefits?</a>

---

There are 1 broken links on playbook/design/templates:

<a href="/disability/how-to-file-claim/">Process pages</a>

---

There are 2 broken links on records/get-military-service-records/reconstruct-records:

<a href="/disability/how-to-file-claim/">VA disability compensation claim</a>
<a href="/disability/how-to-file-claim/">Get instructions for filing a disability claim</a>

---

There are 1 broken links on change-direct-deposit-beta:

<a href="/disability/how-to-file-claim/">file a claim for an increase in benefits</a>

---

There are 1 broken links on change-direct-deposit:

<a href="/disability/how-to-file-claim/">File for a VA disability increase</a>

---

There are 2 broken links on :

<a data-nav-path="Disability-&gt;File for a VA disability increase" href="/disability/how-to-file-claim">File for a VA disability increase</a>
<a data-nav-path="Disability-&gt;File a claim for compensation" href="/disability/how-to-file-claim/">File a claim for compensation</a>

---

There are 1 broken links on va-payment-history:

<a href="/disability/how-to-file-claim/">File for a VA disability increase</a>

---

There are 1 broken links on disability-benefits/apply/form-526-disability-claim:

<a aria-current="page" href="/disability/how-to-file-claim/">File for a disability increase</a>

---

There are 1 broken links on disability/about-disability-ratings-beta:

<a href="/disability/how-to-file-claim" id="mpdrwcr">
                                File for increased disability
                            </a>

---

There are 1 broken links on disability/about-disability-ratings:

<a href="/disability/how-to-file-claim" id="fisbhxlem">
                                File for increased disability
                            </a>

---

There are 1 broken links on disability/after-you-file-claim:

<a href="/disability/how-to-file-claim" id="emkcxdzrme">
                                File for increased disability
                            </a>

---

There are 1 broken links on disability/eligibility:

<a href="/disability/how-to-file-claim" id="stlbocaov">
                                File for increased disability
                            </a>

---

There are 1 broken links on disability/file-an-appeal:

<a href="/disability/how-to-file-claim" id="linrubl">
                                File for increased disability
                            </a>

---

There are 1 broken links on disability/get-help-filing-claim:

<a href="/disability/how-to-file-claim" id="nwhthpa">
                                File for increased disability
                            </a>

---

There are 2 broken links on disability/add-remove-dependent:

<a href="/disability/how-to-file-claim" id="fvuvnmknmb">
                                File for increased disability
                            </a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/how-to-file-claim-changed:

<a href="/disability/how-to-file-claim" id="dechkjpasi">
                                File for increased disability
                            </a>

---

There are 2 broken links on disability:

<a href="/disability/how-to-file-claim/" class="no-external-icon">
        <span class="hub-page-link-list__header">
         How to file a claim
          <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
        </span>
        <p class="hub-page-link-list__description">
        Find out how to prepare and file a claim for disability compensation online or by phone or mail.
        </p>
      </a>
<a href="/disability/how-to-file-claim" class="no-external-icon">
        <span class="hub-page-link-list__header">
         File for a VA disability increase
          <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
        </span>
        <p class="hub-page-link-list__description">
        If your service-connected disability has gotten worse, find out how to file a claim to increase your disability rating.
        </p>
      </a>

---

There are 1 broken links on disability/upload-supporting-evidence:

<a href="/disability/how-to-file-claim" id="tgkeflo">
                                File for increased disability
                            </a>

---

There are 2 broken links on disability/va-claim-exam:

<a href="/disability/how-to-file-claim" id="vwoshwsqn">
                                File for increased disability
                            </a>
<a href="/disability/how-to-file-claim/#days-to-complete-claim">See our most recent estimate for the average number of days to complete a claim</a>

---

There are 1 broken links on disability/about-disability-ratings/effective-date:

<a href="/disability/how-to-file-claim/" id="ramul">
                <h4 class="va-nav-linkslist-title">How to apply for disability benefits</h4>

                <p class="va-nav-linkslist-description">Apply online now, or find out how to file a claim in person, by mail, or with the help of a trained professional.</p>
              </a>

---

There are 2 broken links on disability/eligibility/illnesses-within-one-year-of-discharge:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/" id="elzrpe">
                <h4 class="va-nav-linkslist-title">How to apply for disability benefits</h4>

                <p class="va-nav-linkslist-description">File a claim online now, or find out how to file a claim in person, by mail, or with the help of a trained professional.</p>
              </a>

---

There are 1 broken links on disability/eligibility/ptsd:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/former-pows:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/special-claims/1151-claims-title-38:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/special-claims/automobile-allowance-adaptive-equipment:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/special-claims/birth-defects:

<a href="/disability/how-to-file-claim">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/special-claims/clothing-allowance:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/special-claims/temporary-increase-after-surgery-or-cast:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/special-claims/temporary-rating-prestabilization:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/special-claims/temporary-increase-for-time-in-hospital:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/special-claims/unemployability:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 2 broken links on disability/eligibility/hazardous-materials-exposure/agent-orange:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/asbestos:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/camp-lejeune-water-contamination:

<a href="/disability/how-to-file-claim/">File a claim online on VA.gov</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/gulf-war-illness-afghanistan:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/gulf-war-illness-southwest-asia:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/ionizing-radiation:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/mustard-gas-lewisite:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/project-112-shad:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/specific-environmental-hazards:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/agent-orange/c-123-aircraft:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 5 broken links on disability/eligibility/hazardous-materials-exposure/agent-orange/navy-coast-guard-ships-vietnam:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/agent-orange/registry-health-exam:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/agent-orange/related-diseases:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 2 broken links on disability/eligibility/hazardous-materials-exposure/agent-orange/service-in-vietnam-korea:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/agent-orange/testing-storage-areas:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/eligibility/hazardous-materials-exposure/agent-orange/thailand-military-bases:

<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 4 broken links on disability/how-to-file-claim/additional-forms:

<a href="/disability/how-to-file-claim">
          How to file claim
      </a>
<a href="/disability/how-to-file-claim/">File a disability compensation claim</a>
<a href="/disability/how-to-file-claim/">File a disability compensation claim</a>
<a href="/disability/how-to-file-claim/">File a disability compensation claim</a>

---

There are 1 broken links on disability/how-to-file-claim/evidence-needed:

<a href="/disability/how-to-file-claim">
          How to file claim
      </a>

---

There are 6 broken links on disability/how-to-file-claim/when-to-file:

<a href="/disability/how-to-file-claim">
          How to file claim
      </a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 2 broken links on disability/how-to-file-claim/evidence-needed/fully-developed-claims:

<a href="/disability/how-to-file-claim">
          How to file claim
      </a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 2 broken links on disability/how-to-file-claim/evidence-needed/standard-claims:

<a href="/disability/how-to-file-claim">
          How to file claim
      </a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 3 broken links on disability/how-to-file-claim/when-to-file/pre-discharge-claim:

<a href="/disability/how-to-file-claim">
          How to file claim
      </a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>
<a href="/disability/how-to-file-claim/">Find out how to file a claim for disability compensation</a>

---

There are 1 broken links on disability/how-to-file-claim/when-to-file/pre-discharge-claim/file-while-overseas:

<a href="/disability/how-to-file-claim">
          How to file claim
      </a>

---
```